namespaces/live-1.cloud-platform.service.justice.gov.uk/token-verification-api-dev/resources/elasticache.tf

```diff
@@ -13,6 +13,8 @@ module "tva_elasticache_redis" {
   team_name              = var.team_name
   number_cache_clusters  = var.number_cache_clusters
   node_type              = "cache.t2.small"
+  engine_version         = "5.0.6"
```

Comment:
upgrade to latest - not in active use yet so don't mind if there's downtime

Reply:
It's possible that applying this will recreate your instance - i.e. losing any data you have stored in the current one. If that happened, would it be a problem?

Reply:
thanks for the heads up, no it will be fine